src/planter/planter.controller.ts

```
@@ -3,29 +3,47 @@ import { PlanterService } from './planter.service';
 @UseGuards(AuthJWTGuard)
 @ApiBearerAuth()
 export class PlanterController {
   constructor(private readonly planterService: PlanterService) {}
 
   @ApiOperation({
     summary: "유저 식물들 정보 API",
-    description: "유저가 키웠던 식물 정보들을 반환합니다"
+    description: "유저의 모든 식물 정보를 반환하는 API 입니다."
   })
-  @Get()
+  @Get("/all-user-plants")
```

ESLint

[error] Replace `"/all-user-plants"` with `'/all-user-plants'`

(prettier/prettier)